ts/features/bonus/cgn/screens/activation/CgnActivationCompletedScreen.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { View } from "native-base";
+import { GlobalState } from "../../../../../store/reducers/types";
+import { Dispatch } from "../../../../../store/actions/types";
+import { H1 } from "../../../../../components/core/typography/H1";
+import BaseScreenComponent from "../../../../../components/screens/BaseScreenComponent";
+
+type Props = ReturnType<typeof mapStateToProps> &
+  ReturnType<typeof mapDispatchToProps>;
+
+/**
+ * Screen which is displayed when a user requested a CGN activation
+ * and it took too long to get an answer from the server
```

Comment:
Is this description relative to this screen?
It seems to refer `CgnActivationTimeoutScreen.tsx`

Reply:
Yes i noticed and fixed in the detailed PR about the UI of single screens [here the reference](https://github.com/pagopa/io-app/pull/2758/files#diff-06755a41d358c1feb615b6264127baff2a4fae73f0b389bdef65477339fde5c0R19)